pypeit/core/flux_calib.py

```diff
@@ -660,34 +676,35 @@ def get_sensfunc_factor(wave, wave_zp, zeropoint, exptime, tellmodel=None, extin
     Get the final sensitivity function factor that will be multiplied into a spectrum in units of counts to flux calibrate it.
     This code interpolates the sensitivity function and can also multiply in extinction and telluric corrections.
 
-    FLAM, FLAM_SIG, and FLAM_IVAR are generated
-
-    Args:
-        wave (float `numpy.ndarray`_): shape = (nspec,)
-           Senstivity
-        wave_zp (float `numpy.ndarray`_):
-           Zerooint wavelength vector shape = (nsens,)
-        zeropoint (float `numpy.ndarray`_): shape = (nsens,)
-           Zeropoint, i.e. sensitivity function
-        exptime (float):
-        tellmodel (float  `numpy.ndarray`_, optional): shape = (nspec,)
-           Apply telluric correction if it is passed it. Note this is deprecated.
-        extinct_correct (bool, optional)
-           If True perform an extinction correction. Deafult = False
-        airmass (float, optional):
-           Airmass used if extinct_correct=True. This is required if extinct_correct=True
-        longitude (float, optional):
-            longitude in degree for observatory
-            Required for extinction correction
-        latitude:
-            latitude in degree for observatory
-            Required  for extinction correction
-        extrap_sens (bool, optional):
-            Extrapolate the sensitivity function (instead of crashing out)
+    Parameters
+    ----------
+    wave: `numpy.ndarray`
```

Comment:
`numpy.ndarray` --> `numpy.ndarray`_ for a few of these

Reply:
done